.github/actions/setup-macos/action.yml

```diff
@@ -19,6 +19,17 @@ outputs:
 runs:
   using: "composite"
   steps:
+    # github actions overwrites brew's python. Force it to reassert itself, by running in a separate step.
+    # Borrowed from https://github.com/mesonbuild/meson/blob/master/.github/workflows/macos.yml#L87-L92
```

Comment:
`blob/master` will change. It would be better to add a stable commit with links like this.

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/2224